src/main/kotlin/org/malachite/estella/services/UserService.kt

```diff
@@ -56,12 +57,31 @@ class UserService(
             firstName = user.firstName,
             lastName = user.lastName
         )
+        // TODO [ES-187] - It won't work because of encryption on setter (it invokes encryption of encrypted password)
         updated.password = user.password
```

Comment:
To działa ponieważ user nie jest typu User, a typu UserRequest, przez co w momencie setowania hasło w postaci plain_text jest hashowane i ustawiane w updated userze. O ile pamiętam nawet @Davenury to otestował

Reply:
Dobra xD, zmyliła mnie nazwa user, głupotę napisałem, późno w nocy było xd

Reply:
Ale już nawiązując to nie wiem czy dobrym pomysłem jest robienie tego tym setterem kotlinowym, bo znak '=' zaciemnia co się tak naprawdę dzieje (setPassword by już było lepsze, ale chyba nie da się explicitly go użyć :/)

Reply:
Nie jestem pewny, czy to otestowałem, ale pamiętam, że było z tym kilka problemów i siedzieliśmy z Radkiem i to ogarnialiśmy - I faktycznie był problem z tym na testach, ale to działa sprawnie. (chyba przy zmianie hasła to było, albo coś takiego)